Adds ethics/values statement (#1)
Rhodes-CS-Department/website: ethics-and-diversity.md

In the Computer Science program at Rhodes, we take the last statement of the
vision especially seriously. We recognize that computer science and the
technologies enabled by computers have enormous impact on society. Computer
scientists have the ability to enable rapid, large-scale change—change that can
help society and individuals or can reinforce structures that marginalize and
impose inequality.

As such, we are firmly committed to creating computer scientists that recognize
their value __and responsibility__ to society.

Trained computer scientists have specialized abilities, knowledge, and
positions. As a result, this specialization requires that computer scientists
have a duty to society, just as do many other trained specialists like medical
doctors.

As in the Association for Computing Machinery’s (ACM) [Code of
Ethics](https://www.acm.org/code-of-ethics), this duty starts with a commitment
to use your training as a computer scientist for the betterment of society and
world, recognizing your potential for positive contributions and for harm.
Rhodes Computer Science students are trained to be actively ethical in their

> practice, in that ethical considerations as core to their approach to the

Reviewer: ```suggestion
practice, taking ethical considerations as core to their approach to the
```
